src/app/chat/views/communities.nim

```diff
@@ -342,6 +345,12 @@ QtObject:
       return "Error declining request to join the community"
     return ""
 
+  proc requestCommunityInfo*(self: CommunitiesView, communityId: string) {.slot.} =
+    try:
+      self.status.chat.requestCommunityInfo(communityId)
+    except Exception as e:
+      error "Error fetching community info", msg = e.msg
```

Comment:
should the UI display an error?

Reply:
I'm not sure where we'd show it. The fetching is very much "in the background". Maybe we'd need some kind of "error center" where those kinds of errors are logged?